Makefile

```diff
@@ -128,7 +128,7 @@ unit-test:
 
 # This override is needed to bring in this PR:
 # https://github.com/operator-framework/operator-sdk/pull/1470
-update-generated: export OPERATOR_SDK_VERSION := v0.10.0
+update-generated: export OPERATOR_SDK_VERSION := v0.16.0
```

Comment:
Remove the comments above.

Reply:
There is also a section in operator-sdk-common.sh where `0.8.x` is mentioned. Let's use that and remove this override.
It will allow flexibility in using other versions.

Reply:
@umangachapagain just to make sure: in operator-sdk-common.sh leave it like this (maybe without the comment)?
```
# Freeze on this version until the following PR makes it into a release:
# https://github.com/operator-framework/operator-sdk/pull/2017
---  OPERATOR_SDK_VERSION="${OPERATOR_SDK_VERSION:-v0.8.2}"
+++. OPERATOR_SDK_VERSION="${OPERATOR_SDK_VERSION:-v0.16.0}"
```
and to remove this export from here completely?

Reply:
Yes.

Reply:
Done